src/main/java/com/testaccounting/data/entity/Employee.java

```diff
@@ -0,0 +1,38 @@
+package com.testaccounting.data.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
+
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(indexes = {
+    @Index(name = "employee_name", columnList = "employeeName", unique = true)
+})
+public class Employee {
+    public Employee(String employeeName) {
```

Comment:
instead of constructors you can use  @NoArgsConstructor and  @AllArgsConstructor annotations